packages/tracing-internal/src/node/integrations/express.ts

```diff
@@ -543,7 +543,9 @@ export function preventDuplicateSegments(
   reconstructedRoute?: string,
   layerPath?: string,
 ): string | undefined {
-  const originalUrlSplit = originalUrl?.split('/').filter(v => !!v);
+  // filter query params
+  const normalizeURL = originalUrl?.split('?')?.[0];
```

Comment:
You can import and use `stripUrlQueryAndFragment` from `@sentry/utils` for this.